File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height:");

    }
    while (n < 1 || n > 8);
    int count = 1;
    for (int i = 0; i < n ; i++)
    {
        for (int k = 0 ; k < n - count; k++)
        {
            printf(" ");
        }
        for (int j = 0; j < count ; j++)
        {
            printf("#");

        }
        printf("  ");


        for (int x = 0 ; x < count ; x++)
        {
            printf("#");
        }
        printf("\n");
        count ++ ;
    }


}

File: pset6/mario.py
n = 0

while True:
  n = input("Height:\n")
  if n.isdigit() == False:
    n = input("Height:\n")
  else :
    n = int(n)
    if n <1 or n >8:
      n = input("Height:\n")
    else :
      break;


count = 1

for i in range(n):
  for k in range(n-count):
    print(" " , end = '')
  for j in range(count):
    print("#" , end = '')
  print("  " , end = '')
  for x in range(count):
    print("#" , end = '')
  print("")
  count += 1
